Core/include/Acts/Vertexing/Vertex.ipp

```diff
@@ -56,6 +55,11 @@ Acts::Vector4& Acts::Vertex<input_track_t>::fullPosition() {
   return m_position;
 }
 
+template <typename input_track_t>
+const Acts::Vector4& Acts::Vertex<input_track_t>::fullSeedPosition() const {
+  return m_seedPosition;
+}
```

Comment:
this will conflict with your refactor @paulgessinger . any preference on what should go in first?

Reply:
Can we get this in? Any objections @paulgessinger?

Reply:
Fixed the merge conflict